diff --git a/tika-core/src/test/java/org/apache/tika/language/TestNGramProfile.java b/tika-core/src/test/java/org/apache/tika/language/TestNGramProfile.java
index 8c234a249..ba68b18bd 100644
--- a/tika-core/src/test/java/org/apache/tika/language/TestNGramProfile.java
+++ b/tika-core/src/test/java/org/apache/tika/language/TestNGramProfile.java
@@ -23,8 +23,6 @@ import java.util.List;
 
 import junit.framework.TestCase;
 
-import org.apache.tika.language.NGramProfile.NGramEntry;
-
 public class TestNGramProfile extends TestCase {
 
     String tokencontent1 = "testaddtoken";
